### PR TITLE
use gamla instead of toolz + remove get from function names

### PR DESCRIPTION
**Pull request by davidstrouk**

### CODE REVIEW

File: cloud_utils/cache/file_store.py [hunk 1]
```diff
@@ -20,27 +19,27 @@ def open_file(mode: Text):
...
-        _hash_to_filename,
+        utils.hash_to_filename,
         storage.download_blob_as_string(bucket_name),
         json.loads,
     )
 
 
-def get_local_path_for_hash(object_hash: Text) -> pathlib.Path:
-    local_path = _LOCAL_CACHE_PATH.joinpath(_hash_to_filename(object_hash))
+def local_path_for_hash(object_hash: Text) -> pathlib.Path:
```
Comment (erez-work) [inline cloud_utils/cache/file_store.py:37]: Is this a public function (is it used in depended repos?)

File: cloud_utils/cache/mongo.py [hunk 2]
```diff
@@ -2,28 +2,27 @@
...
-import toolz
-from toolz import curried
-
 
 ASCENDING = pymongo.ASCENDING
 DESCENDING = pymongo.DESCENDING
 
 
-def get_client(mongodb_uri: Text, **kwargs):
+def client(mongodb_uri: Text, **kwargs):
```
Comment (erez-work) [inline cloud_utils/cache/mongo.py:10]: This is definitely public, let's change in depended repos

File: cloud_utils/cache/motor.py [hunk 3]
```diff
@@ -3,18 +3,17 @@
 import gamla
 import pymongo
 from motor import motor_asyncio
-from toolz import curried
 
 
-def get_database_collection(
+def database_collection(
```
Comment (erez-work) [inline cloud_utils/cache/motor.py:8]: Same here

File: setup.cfg [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = cloud-utils
-version = 0.0.1
+version = 2
```
Comment (erez-work) [inline setup.cfg:3]: hmm, why the change here?

**Review by erez-work — APPROVED** [review]

File: setup.cfg (hunk 4 above)
Comment (davidstrouk) [reply setup.cfg:3]: I would like to specify the package version when importing it in requirements.txt, as we do with gamla.
This way when inserting changes such as function name modified we can update the version of the package in the projects that use it, and have control over the version.

Comment (davidstrouk) [reply setup.cfg:3]: Actually I'm seeing that we use the commit hash as version, so I will keep the same version number.
